cartex/CMakeLists.txt

```diff
@@ -54,15 +54,22 @@ function(compile_as_cuda)
   endif()
 endfunction()
 
+# ---------------------------------------------------------------------
+# generate config file
+# ---------------------------------------------------------------------
+configure_file(${CMAKE_CURRENT_SOURCE_DIR}/cmake/cmake_config.inc.in 
+    ${CMAKE_CURRENT_BINARY_DIR}/include/cmake_config.inc)
```

Comment:
maybe we can move this bit into `src/common/CMakeLists.txt`?

Reply:
done